src/pyscipopt/scip.pxi

```diff
@@ -3543,6 +3545,7 @@ cdef class Model:
     def presolve(self):
         """Presolve the problem."""
         PY_SCIP_CALL(SCIPpresolve(self._scip))
+        self._bestSol = Solution.create(self._scip, SCIPgetBestSol(self._scip))
```

Comment:
this might be the most "dangerous" change, but I don't think it causes problems.

Reply:
I don't think it will cause any problems either. We have the same thing after `optimize`, so it should just be `NULL` in the case where it does nothing